mobile/src/feeds/home-feeds-screen.tsx

```diff
@@ -67,7 +69,9 @@ class HomeFeedsScreenImpl extends React.Component<FeedsScreenProps & FeedScreenD
     }
 
     public componentDidMount() {
-        this.props.fetchNewFeeds();
+        if (this.props.user) {
```

Comment:
i think not. you need to subscribe to event where you load new users. This way it'll check for user (which might be empty) and won't load anything until you scroll